router/initoutputs_test.go

```diff
@@ -192,6 +192,24 @@ func TestBuildAndInitOtpt(t *testing.T) {
 			false,
 			"*outputs.ServiceNowOutput",
 		},
+		{
```

Comment:
Are these changes related to this PR?

Reply:
i added these tests to improve `router/builders.go` coverage in  codecov bot, after removed [these functions](https://github.com/aquasecurity/postee/pull/271/files/71395a40805263bc8a0b74b51aa2168df797768b#diff-48982bfea0eb5136efb328357261341d8167a96ed1b49c8319afad977998a933L92-L97)